### DISCUSSION TITLE
Is there any way to iterate over the atoms and bonds in rdkit js?

### DISCUSSION THREAD

**Discussion by Suryarajan3** [rdkit/rdkit#6006] [Development]
Is there any way to iterate over the atoms and bonds in rdkit js. 

In python, I found the below method GetAtoms() and GetBonds()

// Iterate over the atoms
for i, atom in enumerate(mol.GetAtoms()):

// Iterate over the bonds
for i, bond in enumerate(mol.GetBonds()):

Is there any method available for the same in rdkit js.

Thanks in advance

**Comment by ptosco** [comment]
Currently there are no methods to iterate over atoms and bonds in `rdkit-js`.

**Reply by ShangChien** [reply]
Very eager to see the Chem.RWMol()  functions in rdkitjs

**Comment by Suryarajan3** [comment]
Is there any plan to add these methods in the coming releases?

Thanks in advance
